src/format/opamFile.ml

```diff
@@ -1170,12 +1170,12 @@ end
 module ConfigSyntax = struct
 
   let internal = "config"
-  (* This version is used as a marker for the whole opam root, so it's not
-     strictly speaking the actual format of the config file *)
   let format_version = OpamVersion.of_string "2.1"
```

Comment:
Shouldn't this now be `"2.0"` or do you want to release the field and then change that and opam-state separately?